docs/docs/getting_started/starter_example_local.md

```diff
@@ -21,6 +21,8 @@ To load in a Mistral-7B model just do `ollama pull mistral`
 
 **NOTE**: You will need a machine with at least 32GB of RAM.
 
+To import `llama_index.llms.ollama`, you should run `pip install llama-index-llms-ollama`. More integrations are all listed on https://llamahub.ai.
```

Comment:
Awesome! Also, I think this code requires `pip install llama-index-embeddings-huggingface` for the embedding model

It could also be switched to use Ollama embeddings too tbh

Reply:
Actually, I can fix this in this PR

Reply:
done :)